src-docs/src/views/accessibility/accessibility_example.js

```diff
@@ -75,40 +62,119 @@ export const AccessibilityExample = {
               available to all users.&quot;{' '}
-              <EuiLink
-                href="http://webaim.org/techniques/css/invisiblecontent/"
-                external
-              >
+              <EuiLink href="http://webaim.org/techniques/css/invisiblecontent/">
                 Learn more about invisible content
               </EuiLink>
             </p>
           </EuiCallOut>
-        </div>
+          <EuiSpacer />
+          <p>
+            <em>
+              Using a screen reader, verify that there is a second paragraph.
+            </em>
+          </p>
+        </>
       ),
       props: {
         EuiScreenReaderOnly,
       },
-      snippet: screenReaderOnlySnippet,
+      snippet: `<EuiScreenReaderOnly>
+  <!-- visually hidden content -->
+</EuiScreenReaderOnly>`,
       demo: <ScreenReaderOnly />,
     },
     {
-      title: 'Skip link',
       source: [
         {
           type: GuideSectionTypes.JS,
-          code: skipLinkSource,
+          code: screenReaderButtonSource,
+        },
+      ],
+      text: (
+        <>
+          <h3>Within a button</h3>
+          <p>
+            If the screen reader content is <strong>contained within</strong> a
+            focusable element, you should consider adding{' '}
+            <EuiCode language="scss">{'position: relative;'}</EuiCode> to the
+            focusable element. This will fix any screen reader focus rings to
+            stay within the bounds of the focusable element.
+          </p>
+          <p>
+            <em>
+              Using a screen reader, tab through the following example with your
+              keyboard to verify the focus outline and screen reader text.
+            </em>
+          </p>
+        </>
+      ),
+      props: {
+        EuiScreenReaderOnly,
+      },
+      snippet: `<button style={{ position: 'relative' }}>
+  Button text
+  <EuiScreenReaderOnly>
+    <!-- visually hidden content -->
+  </EuiScreenReaderOnly>
+</button>`,
+      demo: <ScreenReaderButton />,
+    },
+    {
+      source: [
+        {
+          type: GuideSectionTypes.JS,
+          code: screenReaderFocusSource,
         },
+      ],
+      text: (
+        <>
+          <h3>Showing on focus</h3>
+          <p>
+            If the wrapped element <strong>is focusable</strong>, you must use
+            the <EuiCode>showOnFocus</EuiCode> prop to visibly show the element
+            to all users when focused.
+          </p>
+          <p>
+            <em>
+              Tab through the following example with your keyboard to verify the
+              element is visible on focus.
+            </em>
+          </p>
+        </>
+      ),
+      props: {
+        EuiScreenReaderOnly,
+      },
+      snippet: `<EuiScreenReaderOnly showOnFocus>
+  <!-- visually hidden content, displayed on focus -->
+</EuiScreenReaderOnly>`,
+      demo: <ScreenReaderFocus />,
+    },
+    {
+      title: 'Skip link',
+      source: [
```

Comment:
I love the new page sections/organization!! It's a huge improvement and brings a ton of clarity to the previous iteration just IMO 😍